src/utils/rest-api.js

```diff
@@ -409,7 +409,13 @@ export function createCase({ name, description, file, parentDirectoryUuid }) {
     return backendFetch(url, {
         method: 'post',
         body: formData,
-    }).then((response) => handleResponse(response, false));
+    }).then((response) =>
+        response.ok
+            ? response
+            : response.status === 422
```

Comment:
this is not necessary, you already check status code in create-case-dialog.js, we don't need to do it twice